### PR TITLE
add contribution guidelines

### PR DESCRIPTION
**Pull request by itaysk**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Contribution Guide
+
+Thank you for taking interest in contributing to Tracee!  
+
+## Issues
+
+- Feel free to open issues for any reason as long as you make it clear what this issue is about: bug/feature/proposal/comment.
+- For qustions and general discussions, please do not open an issue, and instead create a dicussion in the "Discussions" tab.
```
Comment (yanivagman) [inline CONTRIBUTING.md:8]: typo: qu(e)stions,
typo: di(s)cussion

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- Please spend a minimal amount of time giving due diligence to existing issues or discussions. Your topic might be a duplicate. If it is, please add your comment to the existing one.
+- Please give your issue or discussion a meaningful title that will be clear for future users.
+- The issue should clearly explain the reason for opening, the proposal if you have any, and any relevant technical information.
+- For technical questions, please explain in detail what you were trying to do, provide an error message if applicable, and versions of your environemnt and Tracee that you used.
```
Comment (yanivagman) [inline CONTRIBUTING.md:12]: typo: environment
suggested: Tracee -> Tracee version

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## Pull Requests
+
+1. Every Pull Request should have an associated Issue unless it is a trivial fix.
+1. Your PR is more likely to be accepted if it focuses on just one change.
+1. Describe what the PR does. There's no convention enforced, but please try to be concise and descriptive. Treat the PR description as a commit message. Titles that start with "fix"/"add"/"improve"/"remove" are good examples.
+1. There's no need to add or tag reviewers, if your PR is left unattended for too long, you can add a comment to bring it up to attention, optionally "@" mention one of the maintainers that was involved with the issue.
+1. If a reviewer commented on your code or asked for changes, please remember to mark the discussion as resolved after you address it and re-request a review.
+1. When addressing comments, try to fix each suggestion in a seperate commit.
+1. Tests are not required at this point as Tracee is evolving fast, but if you can include tests that will be appreciated.
```
Comment (yanivagman) [inline CONTRIBUTING.md:22]: Numbering is '1' for all bullets

File: CONTRIBUTING.md (hunk 1 above)
Comment (itaysk) [reply CONTRIBUTING.md]: thanks

File: CONTRIBUTING.md (hunk 3 above)
Comment (itaysk) [reply CONTRIBUTING.md:22]: this is fine in markdown...

**Review by yanivagman — APPROVED** [review]

**Pull request merged**